cedar-dafny/validation/thm/model.dfy

```diff
@@ -758,7 +758,16 @@ module validation.thm.model {
     requires IsDecimalComparisonName(name)
     requires ExtensionFunSafeRequires(name, args)
     ensures ExtensionFunSafeEnsures(name, args)
-  {}
+  {
```

Comment:
This is greater than 10M according to my VSCode and the internal dashboard but doesn't show up in https://github.com/cedar-policy/cedar-spec/pull/56 for some reason?

Reply:
The internal dashboard is (currently) based on the internal code, so there may be differences there. As far as the difference between VSCode and the metric used for #56 (which is outlined under "Checking Proof Stability" in the README) -- likely just due to different default settings between the command line build and VSCode. 

@emina may know more.

Reply:
That and there are some differences between RU counts between VSCode and the command line even when the same flags are used.  We should use the command line counts as the ground truth.